lib/spack/spack/util/azure_blob.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+"""This file contains the definition of the Azure Blob storage Class used to
+   integrate Azure Blob storage with spack buildcache.
+"""
```

Comment:
Would it make sense to call this file just `azure.py`? Asking since I don't know much the domain so I might be missing details why it's better not to.

Reply:
azure may be too general? azure_blob is specific to azure blob storage only.